MacOSExample/package.json

```diff
@@ -3,26 +3,27 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "android": "react-native run-android",
-    "ios": "react-native run-ios",
-    "macos": "react-native run-macos",
+    "macos": "react-native-macos run-macos",
     "lint": "eslint .",
-    "start": "react-native start",
+    "start": "react-native-macos start",
     "test": "jest"
   },
   "dependencies": {
+    "@react-native-async-storage/async-storage": "^1.24.0",
     "@react-navigation/native": "^6.1.6",
     "@react-navigation/native-stack": "^6.9.12",
     "@react-navigation/stack": "^6.3.17",
     "hoist-non-react-statics": "^3.3.2",
     "invariant": "^2.2.4",
+    "patch-package": "^8.0.0",
+    "postinstall-postinstall": "^2.1.0",
     "react": "18.2.0",
-    "react-native": "0.72.4",
+    "react-native": "^0.73.9",
     "react-native-gesture-handler": "link:../",
-    "react-native-macos": "^0.72.0",
-    "react-native-reanimated": "^3.5.1",
-    "react-native-safe-area-context": "^4.7.2",
-    "react-native-screens": "^3.25.0"
+    "react-native-macos": "^0.73.35",
```

Comment:
Why not 0.74?

Reply:
There are multiple breaking changes in `0.74` that break `RNGH`, `Rea`, and bundle loading.

I managed to fix most of them but i'm still working on it.